frontend/src/components/Header/Header.jsx

```diff
@@ -5,27 +5,39 @@ import { Link } from "react-router-dom";
 import { connect } from 'react-redux';
 import './Header.styles.sass'
 
-function Header(props) {
+function Header({userName, isAdmin}) {
   return (
-    <nav className="NavbarItems" >
-      <img src={FoxticketLogo} alt='navigation bar logo'></img>
-      <h3 className="nav-text">Foxticket</h3>
-      <ul className="nav-menu">
-      { 
-        props.type ==='admin' &&
-         <Link to="/admin" className="nav-links" >Admin</Link>
-      }
-        <li className="nav-links">{props.userName}</li>
-        <Logout />
-      </ul>
-    </nav>
+    <>
+    { localStorage.getItem('token') &&
```

Comment:
@tamas-w let's have always a header not just when there is something in the localstorage